### PR TITLE
Add timer settings

### PR DESCRIPTION
**Pull request by Evomatic**
## Goal
Related issue: https://github.com/webdevbroz/pomodoro-app/issues/23

We want users to have the ability to change the length of their designated `pomodoro`, `short break`, and `long break` timed sessions. This PR adds a module in the settings menu for doing just that.
## How to test
- Change the different time sessions and make sure the time updates and the circle properly ticks down according to the selected time.
- Check the styling against the figma file. Make sure the layouts work for desktop and mobile.

## Screenshots
mobile:
image

desktop:
image

### CODE REVIEW

File: src/components/pomodoro/pomodoro-settings.tsx [hunk 1]
```diff
@@ -40,14 +47,63 @@ export default function PomodoroSettings(): ReactElement {
...
+        <PomodoroTimerSettings onTimerChange={handleTimeChange} />
         <PomodoroColourSettings onColourChange={handleColourChange} />
         <PomodoroFontSettings onFontChange={handleFontChange} />
         <Button onClick={handleApplyNewSettings}>Apply</Button>
       </DialogContent>
     </Dialog>
   );
 }
+
+
```
Comment (blue2wo) [inline src/components/pomodoro/pomodoro-settings.tsx:63]: Are we needing this anymore?

File: src/components/pomodoro/pomodoro-settings.tsx [hunk 2]
```diff
@@ -40,14 +47,63 @@ export default function PomodoroSettings(): ReactElement {
...
+//       <br />
+//       <label>
+//         Seconds:
+//         <input type="number" value={timerData.seconds} onChange={handleSecondsChange} />
+//       </label>
+//     </div>
+//   );
+// };
+
+// export default PomodoroTimerSettings;
```
Comment (blue2wo) [inline src/components/pomodoro/pomodoro-settings.tsx:109]: Are we needing this anymore?

Comment (Evomatic) [reply src/components/pomodoro/pomodoro-settings.tsx:109]: @blue2wo oh nope! I will remove that

**Review by blue2wo — APPROVED** [review]
LGTM 🌮 🦖

**Pull request merged**